src/tools/compiletest/src/runtest.rs

```diff
@@ -3824,6 +3824,7 @@ impl<'test> TestCx<'test> {
             .arg(format!("-Ldependency={}", &support_lib_deps_deps.to_string_lossy()))
             .arg("--extern")
             .arg(format!("run_make_support={}", &support_lib_path.to_string_lossy()))
+            .arg("--edition=2021")
```

Comment:
Hmm, do we want to force edition 2021 for *every* rmake test?

Reply:
I don't see why not. The `rmake` test files are “developer-facing” after all unlike the Rust files found in e.g. `tests/ui`. They are test runners, not “test subjects”. Lol, I'm lacking words. Similarly, `compiler/` / `rustc_*` crates all default to Rust 2021.

Reply:
Make sense yeah, I was just wondering if where was any weird edge cases I have not considered